modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/ScannableTableImpl.java

```diff
@@ -221,7 +221,7 @@ public <RowT> Publisher<RowT> indexLookup(
     @Override
     public <RowT> CompletableFuture<RowT> primaryKeyLookup(
             ExecutionContext<RowT> ctx,
-            @Nullable InternalTransaction explicitTx,
+            InternalTransaction tx,
```

Comment:
I think, it worth to add assertion making sure no-one is passing `null` instead of transaction

Reply:
Done